src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -144,23 +145,31 @@ internal static ScriptHostConfiguration CreateScriptHostConfiguration(WebHostSet
                 LoggerFactoryBuilder = settings.LoggerFactoryBuilder
             };
 
+            if (inStandbyMode)
+            {
+                scriptHostConfig.FileLoggingMode = FileLoggingMode.Always;
+                scriptHostConfig.HostConfig.StorageConnectionString = null;
+                scriptHostConfig.HostConfig.DashboardConnectionString = null;
+                scriptHostConfig.RootScriptPath = Path.Combine(Path.GetTempPath(), "Functions", "Standby");
+            }
+
             scriptHostConfig.HostConfig.HostId = Utility.GetDefaultHostId(_settingsManager, scriptHostConfig);
 
             return scriptHostConfig;
         }
 
-        private static void InitializeFileSystem(string scriptPath)
+        private static void InitializeFileSystem()
         {
             if (ScriptSettingsManager.Instance.IsAzureEnvironment)
             {
-                // When running on Azure, we kick this off on the background
                 Task.Run(() =>
                 {
                     string home = ScriptSettingsManager.Instance.GetSetting(EnvironmentSettingNames.AzureWebsiteHomePath);
                     if (!string.IsNullOrEmpty(home))
                     {
                         // Delete hostingstart.html if any. Azure creates that in all sites by default
-                        string hostingStart = Path.Combine(scriptPath, "hostingstart.html");
+                        string siteRootPath = Path.Combine(home, @"site\wwwroot");
```

Comment:
clarifying this - we always want to delete from the users wwwroot, not the striptRoot (which may be in tmp dir)

Reply:
Right. I deleted the parameter to make this clearer. This code was relying on the fact that we default the script root to wwwroot, though they aren't required to be the same. This new code is more correct. The hostingstart.html file is created in the Antares wwwroot.